packages/sdk/tests/e2e/dataProtectorCore/grantAccess.test.ts

```diff
@@ -98,6 +98,25 @@ describe('dataProtectorCore.grantAccess()', () => {
     MAX_EXPECTED_WEB2_SERVICES_TIME
   );
 
+  it(
+    'Should be able to authorize specific user when `any` is already authorized',
+    async () => {
+      // grantAccess to any user
+      await dataProtectorCore.grantAccess({
+        ...input,
+        authorizedUser: ethers.ZeroAddress,
+        authorizedApp: sconeAppAddress,
+      });
+      // grantAccess to specific user
+      await dataProtectorCore.grantAccess({
+        ...input,
+        authorizedApp: sconeAppAddress,
+      });
+      expect(true).toBe(true);
```

Comment:
you need to create a specific user and pass it as a parameter to later test if this user is in the grantedAccess list.

let test = expect(true).toBe(true); is always positive.

Reply:
`expect(true).toBe(true)` is always true, but it doesn't necessarily mean that the test will always succeed. Here, what we want to test is that the second call to the `grantAccess` function that does not revert. 

We use this quick win multiple times in [other tests as well](https://github.com/iExecBlockchainComputing/dataprotector-sdk/blob/dc1ae28835df9ec2462461cb3490e2544d705103/packages/sdk/tests/e2e/dataProtectorCore/revokeAllAccess.test.ts#L93).

However, it's true that we can also test the result of the function => Commit: 865291466d2189c08d03d9c3e6a08f737d56214a